### PR TITLE
PRF Changes

### PR DESCRIPTION
**Pull request by abaicus**
### Summary
- Changes plugin name;
- Deprecates the old `Page templates` tab inside the Template cloud inserter modal;
- Now, only users with a valid library license can see the block/elementor/beaver functionalities;
- Valid licenses are either Template Cloud dedicated ones, or Neve licenses from tier 3;
- Migrates old users to the new experience, depending on the active license, and conditioned by the templates they previously had saved in the cloud according to [this comment here](https://github.com/Codeinwp/templates-cloud/issues/91#issuecomment-2467982241).

### Will affect visual aspect of the product

YES (but only on the backend)

### Test instructions

- Follow the flows mentioned in this comment and confirm that the plugin acts accordingly: https://github.com/Codeinwp/templates-cloud/issues/91#issuecomment-2467982241
- Except from what is mentioned there, when the user doesn't have neve, we still show the import buttons, except that when the user tries importing a starter site, they will be prompted to install the theme as well. 

Arc 2024-11-25 15 44 30

## Check before Pull Request is ready:

* [x] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

Closes Codeinwp/templates-cloud#91.

### CODE REVIEW

File: beaver/src/export.js [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
+	<h1>${ window.tiTpc.exporter.modalLabel }</h1>
+	<label for="tpc-${ node }">${ window.tiTpc.exporter.textLabel }</label>
+	<input id="tpc-${ node }" type="text" placeholder="${ window.tiTpc.exporter.textPlaceholder }" />
+</div>`;
+
+	FLBuilder.confirm( {
+		message,
+		ok: () => {
+			const input = document.getElementById( `tpc-${ node }` );
+			const title = input.value || 'Template';
```
Comment (Soare-Robert-Daniel) [inline beaver/src/export.js:19]: Can we also have this default value as a translatable string via `window.tiTpc.exporter`?

File: beaver/src/components/header.js [hunk 2]
```diff
@@ -85,10 +83,7 @@ const Header = ( { closeModal, getOrder, getSearchQuery } ) => {
 							/>
 						) }
 
-						{ window.tiTpc.postTypes.includes(
-							window.tiTpc.postType
-						) &&
-							parseInt( window.tiTpc.tier ) === 3 && (
+						{ window.tiTpc.postTypes.includes(window.tiTpc.postType) && (
```
Comment (Soare-Robert-Daniel) [inline beaver/src/components/header.js:86]: One small suggestion here: you can declare this as a constant outside the component with a name like `isPostTypeCompatible` to highlight the compatibility check.

**Comment by abaicus** [comment]
@ineagu It should work with all previously included licenses. It uses the same license mechanism we previously did on TPC so nothing should be different.

**Comment by ineagu** [comment]
As a new user, after plugin activation, I still see the menu item for Templates Cloud, once I click on it or somewhere else, it goes away.
Screenshot 2024-11-27 at 12 54 16

@abaicus

**Pull request merged**